appveyor.yml

```diff
@@ -51,6 +51,7 @@ build_script:
   - cd dokan
   - go test -i
   - go test -c
+  - dokan.test
```

Comment:
looks like we were building the dokan test but not running? @zanderz was there a reason?